pkg/daemon/daemon.go

```diff
@@ -519,7 +522,8 @@ func servePrometheusMetrics(configParams *config.Config) {
 			}
 		}
 		http.Handle("/metrics", promhttp.Handler())
-		err := http.ListenAndServe(fmt.Sprintf(":%v", configParams.PrometheusMetricsPort), nil)
```

Comment:
Have you verified that `[]:<port>` is valid?

Reply:
Yes, it works.